tests/python_package_test/test_engine.py

```diff
@@ -1545,6 +1545,59 @@ def test_refit():
     assert err_pred > new_err_pred
 
 
+def test_refit_dataset_params():
+    # check refit accepts dataset_params
+    X, y = load_breast_cancer(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=42)
+    lgb_train = lgb.Dataset(X_train, y_train)
+    train_params = {
+        'objective': 'binary',
+        'metric': 'binary_logloss',
+        'verbose': -1,
+        'min_data': 10
```

Comment:
```suggestion
        'min_data': 10,
        'seed': 123
```

Please set a seed here, to remove a source of randomness in the training of the initial Booster.

Reply:
got it!